lib/index.ts

```diff
@@ -459,14 +530,24 @@ export class JellyfishSDK {
 		body: TBody,
 		options?: AxiosRequestConfig,
 	): Promise<AxiosResponse<{ data: TResponse; error: Error }>> {
+		let headers = {};
+
+		if (this.sessionId && this.authToken) {
+			headers = {
+				authorization: `Bearer ${this.sessionId}.${this.authToken}`,
+			};
+		} else if (this.sessionId) {
+			headers = {
+				authorization: `Bearer ${this.sessionId}`,
+			};
+		}
```

Comment:
This logic is copied around multiple times. Would be better to factor it out in a separate method.

Reply:
This was moved to a reusable method. Resolving this and all related threads.